src/Components/Theme.jsx

```diff
@@ -111,6 +111,10 @@ Theme.overrides = {
             fontSize: "20px",
             letterSpacing: "0.1px",
             lineHeight: "28px",
+            [Theme.breakpoints.down("sm")]: {
+                fontSize: "18px",
+                lineHeight: "22px",
```

Comment:
Line height should ideally be in ems in case fontSize ever changes.

Reply:
But I guess in this case it's probably ok to keep since this is for a theme.

Reply:
I have been following the design system from figma. I can work on converting the ratios to ems as a future project I think. The font responsiveness is fairly brittle right now, but passable I think.